lib/list.js

```diff
@@ -0,0 +1,21 @@
+import { getLanguage } from './common';
```

Comment:
Looks like you're missing a `.js` here (according due to the check failure).

```suggestion
import { getLanguage } from './common.js';
```

Reply:
Yep on it - I'm also adding some unit tests.